simulariumio/writers/binary_chunk.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
```

Comment:
BinaryChunk is used to determine how many files to save and how to chunk up the data, if the data is larger than the max file size

Reply:
The above is a perfect line for a comment in the code , right where it says "Object to store size info for a binary file"